### ISSUE TITLE
[BUG] Lottie crashes when starting in Windows

### ISSUE THREAD

**Issue by eduardoagr** [mono/SkiaSharp.Extended#289]
**Description**

When you start an application, with a lottie file, this happens 

<img width="661" alt="image" src="https://github.com/user-attachments/assets/2b0cc9c2-e550-4cea-ab38-3c4373f71472">

**Code**

```
<Grid RowDefinitions="Auto,*,*">
    <Image
        Margin="40"
        Aspect="Center"
        Source="metro_wind_200.png"
        VerticalOptions="Start" />
    <skia:SKLottieView
        Grid.Row="1"
        HeightRequest="300"
        HorizontalOptions="Center"
        Source="metrowind.json"
        VerticalOptions="Start"
        WidthRequest="400">
        <skia:SKLottieView.Behaviors>
            <mct:EventToCommandBehavior
                Command="{Binding GotoHomeCommand}"
                EventName="AnimationCompleted" />
        </skia:SKLottieView.Behaviors>
    </skia:SKLottieView>
    <Label
        Grid.Row="2"
        FontAttributes="Bold"
        FontSize="18"
        HorizontalOptions="Center"
        Text="{x:Static rex:AppResource.Energy}"
        VerticalOptions="Start" />
```

```
        public static MauiApp CreateMauiApp() {
            var builder = MauiApp.CreateBuilder();
            builder
                .UseMauiApp<App>()
#if WINDOWS
                .UseMauiCommunityToolkitMaps("xxx")
#elif ANDROID || IOS
           .UseMauiMaps()
#endif
                .UseSkiaSharp()
```

**Expected Behavior**

The app would start normally

**Actual Behavior**

The app crashes, and tels me this

![image](https://github.com/user-attachments/assets/e2fce82f-264f-485c-85a3-3e0824d8b46d)

**Basic Information**

- Version with issue:  latest
- Last known good version:  latest
- IDE:  Visual Studio 2022 17.11.5
- Platform Target Frameworks: 
  - Windows10.0.19041.0
- Target Devices:   Windows 11
  - Device Windows 11
  
<details>
  <summary>Detailed IDE/OS information (click to expand)</summary>
  
```

Net 8
Maui 8.0.1
Edition	Windows 11 Education
Version	23H2
Installed on	‎10/‎24/‎2022
OS build	22631.4317
Experience	Windows Feature Experience Pack 1000.22700.1041.0


```
</details>

**Screenshots**

![image](https://github.com/user-attachments/assets/1dc65c54-90f3-4faf-860a-a6e3b2f8e278)

**Reproduction Link**

(https://github.com/eduardoagr/PopUpError)

**Comment by superwilly1** [comment]
I have the same issue, since updating to Visual Studio 17.11.5 (and .NET 8.0.10, SDK 8.0.403).
In the output window I see this exception:
```
Exception thrown: 'System.NotSupportedException' in WinRT.Runtime.dll
Exception thrown: 'System.Reflection.TargetInvocationException' in System.Private.CoreLib.dll
```
After downgrading Visual Studio (and .NET), it works again.

**Comment by mattleibow** [comment]
I am not sure why it started crashing, but it appears that the latest previews of SkiaSharp fix it.

You can manually install the latest SkiaSharp in your csproj:

```xml
<PackageReference Include="SkiaSharp.Views.Maui.Controls" Version="2.88.9-preview.2.2" />
```

**Comment by superwilly1** [comment]
> I am not sure why it started crashing, but it appears that the latest previews of SkiaSharp fix it.
> 
> You can manually install the latest SkiaSharp in your csproj:
> 
> ```
> <PackageReference Include="SkiaSharp.Views.Maui.Controls" Version="2.88.9-preview.2.2" />
> ```

Yes this appears to fix the issue indeed. The stable version is released which also works: 2.88.9.
I didn't have a reference to SkiaSharp.Views.Maui.Controls yet, but did have a reference to
<PackageReference Include="SkiaSharp.Extended.UI.Maui" Version="2.0.0" />
Which has a dependency on SkiaSharp.Views.Maui.Controls (>= 2.88.7).
When I added the reference to SkiaSharp.Views.Maui.Controls 2.88.9, the app no longer crashes on Windows.
